dynamic-geoassets-api/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/metno/S-ENDA-documentation/dynamic-geoassets-api
+
+go 1.13
+
+require (
```

Comment:
How can we keep up to date with the dependency development? Do you have to come here and manually increase the versions? Does that work in practice?

Reply:
I am not sure about this, but we should figure out a good strategy yes.